install_config/cluster_metrics.adoc

```diff
@@ -108,52 +108,62 @@ secrets:
 API
 ----
 
-Before it can deploy components, the metrics-deployer service account must be
-granted the `edit` permission for the _openshift-infra_ project:
+Before it can deploy components, the *metrics-deployer* service account must be
+granted the `edit` permission for the *openshift-infra* project:
 
-[options="nowrap"]
 ----
-$ oadm policy add-role-to-user edit system:serviceaccount:openshift-infra:metrics-deployer
+$ oadm policy add-role-to-user \
+    edit system:serviceaccount:openshift-infra:metrics-deployer
 ----
```

Comment:
Seems there's two little procedures here. Should they be step 1 and 2 of a procedure?